scripts/package-system.mk

```diff
@@ -14,7 +14,7 @@ diff: ## Diff Helm release against objects in a Kubernetes cluster
 	kubectl get hr -n $(NAMESPACE) $(NAME) -o jsonpath='{.spec.values}' | helm diff upgrade --allow-unreleased --normalize-manifests -n $(NAMESPACE) $(NAME) . -f -
 
 suspend: ## Suspend reconciliation for an existing Helm release
-	flux suspend hr -n $(NAMESPACE) $(NAME)
+	kubectl patch hr -n $(NAMESPACE) $(NAME) -p '{"spec": {"suspend": true}}' --type=merge
```

Comment:
Flux does not tolerate kubectl edits https://fluxcd.io/flux/faq/#why-are-kubectl-edits-rolled-back-by-flux

Reply:
@stefanprodan thank you for your note! I just added PR for fixing this https://github.com/aenix-io/cozystack/pull/101